### ISSUE TITLE
OpenJS Foundation Cross Project Council Meeting 2019-08-06

### ISSUE THREAD

**Issue by mhdawson** [openjs-foundation/cross-project-council#281]
## Time

**UTC Tue 06-Aug-2019 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 06-Aug-2019 11:00 (11:00 AM) |
| US / Mountain | Tue 06-Aug-2019 12:00 (12:00 PM) |
| US / Central  | Tue 06-Aug-2019 13:00 (01:00 PM) |
| US / Eastern  | Tue 06-Aug-2019 14:00 (02:00 PM) |
| London        | Tue 06-Aug-2019 19:00 (07:00 PM) |
| Amsterdam     | Tue 06-Aug-2019 20:00 (08:00 PM) |
| Moscow        | Tue 06-Aug-2019 21:00 (09:00 PM) |
| Chennai       | Tue 06-Aug-2019 23:30 (11:30 PM) |
| Hangzhou      | Wed 07-Aug-2019 02:00 (02:00 AM) |
| Tokyo         | Wed 07-Aug-2019 03:00 (03:00 AM) |
| Sydney        | Wed 07-Aug-2019 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2019-08-06&iso=20190806T18
* or http://www.wolframalpha.com/input/?i=06PM+UTC%2C+Aug+06%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1vBzM66_o89b1QxNZ1scaFgAfsBGe60wWqRXos0sykcY/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* adds a draft charter template [#279](https://github.com/openjs-foundation/cross-project-council/pull/279)
* doc: swapping refs to TAC with CPC [#275](https://github.com/openjs-foundation/cross-project-council/pull/275)
* ELI5: project charters [#207](https://github.com/openjs-foundation/cross-project-council/issues/207)
* Post-Bootstrap work to be managed by CPC [#115](https://github.com/openjs-foundation/cross-project-council/issues/115)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: https://zoom.us/j/920010234
* For those who just want to watch: https://livestream.openjsf.org

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Jonahss** [comment]
Can't make this one, have a meeting.
Progress on Travel Fund: https://github.com/openjs-foundation/cross-project-council/pull/278

**Comment by kborchers** [comment]
Unfortunately I have to miss today due to a meeting conflict as well. I will be sending out a summary of the last board meeting later this week. No progress last week with onboarding tasks for Architect but hope to discuss CoC items this week.

**Comment by joesepi** [comment]
More PRs to consider as a part of our agenda:
- PR #276 doc: adding meeting notes from July
- PR #278 Travel Fund - Promotion to Stage-1
- PR #274 doc: add gov for first term of voting members
- PR #271 Add process for handling reports

More issues to consider as a part of our agenda:
- #280 Define & document a style guide for the foundation's name
- #273 Process to request legal advice for projects
